js/ui/sortable.js

```diff
@@ -236,9 +238,16 @@ const Sortable = Draggable.inherit({
 
         if($scrollable) {
             const offset = $scrollable.offset();
```

Comment:
Can you separate this into a separate method?

For example getScrollableBoundary, which returns {top, bottom, left, right}